tests/test_crud.py

```diff
@@ -237,3 +237,41 @@ class NameCard(db2.Model):
         await Relation.create(bind=engine, timeout=10, name=r)
     r1 = await Relation.get(relations[0], bind=engine, timeout=10)
     assert r1.name == relations[0]
+
+
+async def test_lookup_287(bind):
+    from gino.exceptions import NoSuchRowError
+
+    class Game(db.Model):
+        __tablename__ = 'games'
+        game_id = db.Column(db.String(32), unique=True)
+        channel_id = db.Column(db.String(1), default='A')
+
+    await Game.gino.create()
+    try:
+        game_1 = await Game.create(game_id='1', channel_id='X')
+        game_2 = await Game.create(game_id='2', channel_id='Y')
+        uq = game_1.update(game_id='3')
+        with pytest.raises(TypeError,
+                           match='Model Game has no table, primary key'):
+            await uq.apply()
```

Comment:
minor: `uq` is not used anywhere else, no need to declare as a variable?

Reply:
ah it was to test `update()` doesn't raise